### ISSUE TITLE
Domains as aliases

### ISSUE THREAD

**Issue by Pixelstrudel** [reddec/syno-cli#4]
Script is working flawlessly with DSM7.1 however I would prefer to use a single certificate with multiple domains as aliases (in my case the root + wildcard domain). Any chance of this getting implemented?

**Comment by reddec** [comment]
Yep, this should be pretty straightforward. However, for your specific case (wildcard + non-wildcard) I have to check with the ACME standard is it possible.

**Comment by Jaco1960** [comment]
Same request here, I would like to use a single certificate with 2 wildcard domains like *.internal.domain, *.domain. I manually created such certificates with acme before. It could be as simple as changing the DOMAINS environment variable to accept ";" as the delimiter instead of the ","

**Comment by reddec** [comment]
@Jaco1960 I am not sure that ACME (Let's Encrypt protocol) will let us issue multiple wildcard domains under one CN. I have to double-check :sweat_smile:
